java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -3538,4 +3538,37 @@ public void bitcount(BaseClient client) {
             assertTrue(executionException.getCause() instanceof RequestException);
         }
     }
+
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void setbit(BaseClient client) {
+        String key1 = UUID.randomUUID().toString();
+        String key2 = UUID.randomUUID().toString();
+
+        assertEquals(1, client.sadd(key2, new String[] {"value"}).get());
+        assertEquals(0, client.setbit(key1, 0, 1).get());
```

Comment:
you should set key1 first... no?

Reply:
No need to set the key as it is treated as an empty string.